Update the execute privilege section (#1098)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/ql/administration/security/dbms/execute-syntax.asciidoc

GRANT EXECUTE ADMIN[ISTRATOR] PROCEDURES
    ON DBMS
    TO role[, ...]

> | Enable the specified roles to execute procedures annotated with `@Admin`. The procedures will be executed with elevated privileges.

Reviewer: ```suggestion
| Enable the specified roles to execute procedures annotated with `@Admin`. The procedures are executed with elevated privileges.
```

> | Enable the specified roles to execute procedures annotated with `@Admin`. Will, in addition, enable using elevated privileges when executing the procedures.

Reviewer 2: ```suggestion
| Enable the specified roles to execute procedures annotated with `@Admin`. The procedures will be executed with elevated privileges.
```

GRANT EXECUTE BOOSTED [USER [DEFINED]] FUNCTION[S] name-globbing[, ...]
ON DBMS
TO role[, ...]

> | Enable the specified roles to execute the given functions with elevated privileges.

Reviewer 2: this should still be `user defined`, normal cypher functions don't go through the same code so are not affected.
There are a few built-in functions that are made the same way as the user defined ones, but they don't rely on any privileges so doesn't matter if they actually are boosted or not (e.g. `fromEpoch`)

Author: Didn't we look into this recently (during the show function work) and found that the boosted privileges applied?

Reviewer 2: That is only true for those few functions, the other functions are real expressions instead

e.g. you can't boost shortestPath

Author: I wonder if that means we should update the output column for boosted roles in show functions?
